src/app/features/pages/categories/categories-form/categories-form.component.scss

```diff
@@ -0,0 +1,11 @@
+.mt{
+    margin-top: 20px;
+}
+
+.file-name {
+    margin-left: 1rem; /* or whatever margin you require */
+}
+
+::ng-deep .ck-editor__editable_inline {
+    min-height: 200px;
+}
```

Comment:
Usar ::ng-deep no es una buena prática. Es recomendable customizar los componentes de librerias dentro del global stylesheet.
https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/frameworks/angular.html#setting-the-height-via-a-global-stylesheet